tsconfig.json

```diff
@@ -8,9 +8,11 @@
     "moduleResolution": "node",
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
+    "allowJs": true,
     "sourceMap": true,
     "baseUrl": ".",
     "types": [
+      "node",
```

Comment:
why is this needed ?

Reply:
This adds types from node. In this case it is needed in the i18n.spec.ts:
`const baseDir = path.join(__dirname, '../../../')`